src/backend/sql/gnc-backend-sql.h

```diff
@@ -372,23 +372,23 @@ struct GncSqlColumnInfo
     GncSqlColumnInfo (std::string&& name, GncSqlBasicColumnType type,
                       unsigned int size = 0, bool unicode = false,
                       bool autoinc = false, bool primary = false,
-                      bool null_allowed = false) :
+                      bool not_null = false) :
```

Comment:
You invert the logic of null_allowed to not_null throughout the code, except for the default value. This means a column is now by default allowed to store null values, where it wasn't before. Is this your intention ?